### ISSUE TITLE
Achieving 100% accuracy with Hypertune

### ISSUE THREAD

**Issue by Threynaud** [GoogleCloudPlatform/cloudml-samples#6]
I started using Hypertune of a dataset of mine using almost the same code than the one in the samples from the documentation.
The distributed version works very well for me but with Hypertune I manage to get an objective value (set to accuracy, like the MNIST sample) of 1.0 = 100% which is quite surprising. Note that I use the same config file that in the example and I get such a high accuracy for high learning rates, close to 0.5. 
I thought that the error was on my side but it turns out that there is the same problem with the MNIST example. In the docs, [here](https://cloud.google.com/ml/docs/quickstarts/training), 100% accuracy is also achieved with a very simple network which is very unlikely to happen.

```
...
state: SUCCEEDED
...
trainingOutput:
  completedTrialCount: '10'
  trials:
  - finalMetric:
      objectiveValue: 1.0
      trainingStep: '5006'
    hyperparameters:
      hidden1: '339'
      hidden2: '30'
      learning_rate: '0.49576010451421226'
    trialId: '4'
  - finalMetric:
      objectiveValue: 1.0
      trainingStep: '5009'
    hyperparameters:
      hidden1: '392'
      hidden2: '248'
      learning_rate: '0.49432185726663225'
    trialId: '5'
```

Also, it might not be related at all but I noticed a big discrepancy between metrics on the training and eval sets. You can find the corresponding stack overflow question [here](http://stackoverflow.com/questions/40305141/outputs-of-the-distributed-version-of-mnist-model/40314606#40314606).

Thanks

EDIT: The stack overflow question is actually not related, I think.

**Comment by elibixby** [comment]
Thanks for the report. I need a little more info here. Do you have (1) summaries from during the hptuning job (tensorboard screenshots are fine but you can also add me as a reader to your GCS bucket two and I'll load them up. <my-github-id>@google.com ), and (2) your job config?

**Comment by puneith** [comment]
@Threynaud  Can you please try out Census sample and see if you are running into same issues.

**Comment by elibixby** [comment]
Closing this as inactive
